microsoft-edge/webview2/concepts/data-privacy.md

```diff
@@ -6,69 +6,117 @@ ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
-ms.date: 05/16/2023
+ms.date: 05/31/2023
 ---
 # Data and privacy in WebView2
 
-The WebView2 team tracks data to keep WebView2 secure and up to date, diagnose issues, and improve WebView2. This article is for WebView2 developers.
-WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License. Additionally, WebView2 follows the standards outlined in the [Microsoft Edge Privacy Whitepaper](../../privacy-whitepaper/index.md).
+The WebView2 team tracks data in order to keep WebView2 secure and up to date, diagnose issues, and improve WebView2.  WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License.

```

Comment:
Can we add a link to the Terms and Conditions?

Reply:
need URL for "WebView2 developers consent to this data collection in the WebView2 Runtime Terms and Conditions License."

https://www.bing.com/search?q=WebView2+Runtime+Terms+and+Conditions

I'll check NuGet package site, and in VS, NuGet Package Manager.

Reply:
Likely: 
https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1774.30/License
but that's a version-specific URL

Reply:
closing in favor of new thread https://github.com/MicrosoftDocs/edge-developer/pull/2594/files#r1214885245